python/tests/operations/slice.py

```diff
@@ -1,589 +1,319 @@
 import os
 import unittest
+import warnings
 
 import numpy as np
 
 import equistore.io
 import equistore.operations as fn
-from equistore import Labels, TensorBlock, TensorMap
+from equistore import Labels
 
 DATA_ROOT = os.path.join(os.path.dirname(__file__), "..", "data")
 TEST_FILE = "qm7-spherical-expansion.npz"
 
 
-class TestSlice(unittest.TestCase):
-    """
-    Unit tests for the slice functions.
-    Tests are organised into several 'Test Blocks' as follows:
-        1) Slicing samples dimension of tensors without gradients.
-        2) Slicing samples dimension of tensors with gradients.
-        3) Slicing properties dimension of tensors without gradients.
-        4) Slicing properties dimension of tensors with gradients.
-        5) Slicing samples and properties dimensions simultaneously.
-        6) Testing the type handling of the user-facing slice function.
-        7) Checking issuance of user warnings when empty blocks are
-            created.
-    """
-
-    # TEST BLOCK 1: SLICING SAMPLES WITHOUT GRADIENTS
-
-    def test_slice_samples_tensorblock_no_gradients(self):
-        tensormap = equistore.io.load(
+class TestSliceSamples(unittest.TestCase):
```

Comment:
FYI, you can have multiple classes inheriting from `TestCase` in the same file, that's usually how I structure multiple test blocks in the same file

Reply:
Ok that is much cleaner, thanks!